### PR TITLE
Set location using correct method

### PR DESCRIPTION
**Pull request by sravfeyn**
Fixes a minor bug https://manage.dimagi.com/default.asp?280455#1515230

### CODE REVIEW

File: corehq/apps/users/models.py [hunk 1]
```diff
@@ -436,6 +436,7 @@ class DomainMembership(Membership):
     timezone = StringProperty(default=getattr(settings, "TIME_ZONE", "UTC"))
     override_global_tz = BooleanProperty(default=False)
     role_id = StringProperty()
+    # This should not be set directly but using set_location method only
```
Comment (jmtroth0) [inline corehq/apps/users/models.py]: Is it worth overriding the `.setter` function to require this?

File: corehq/apps/registration/forms.py [hunk 2]
```diff
@@ -406,9 +406,9 @@ def clean(self):
 class AdminInvitesUserForm(RoleForm, _BaseForm, forms.Form):
     # As above. Need email now; still don't need domain. Don't need TOS. Do need the is_active flag,
     # and do need to relabel some things.
-    email       =  forms.EmailField(label="Email Address",
-                                    max_length=User._meta.get_field('email').max_length)
-#    is_domain_admin = forms.BooleanField(label='User is a domain administrator', initial=False, required=False)
+    email = forms.EmailField(label="Email Address",
+                             max_length=User._meta.get_field('email').max_length)
+    # is_domain_admin = forms.BooleanField(label='User is a domain administrator', initial=False, required=False)
```
Comment (jmtroth0) [inline corehq/apps/registration/forms.py]: Can this just be removed?

**Review by jmtroth0 — APPROVED** [review]
couple small questions

File: corehq/apps/registration/forms.py (hunk 2 above)
Comment (sravfeyn) [reply corehq/apps/registration/forms.py]: Sure

**Pull request merged**